doc-content/enterprise-only/installation/ba-platforms-con.adoc

```diff
@@ -1,6 +1,6 @@
 [id='ba-platforms-con_{context}']
 
-= Available installation platforms
+= {PRODUCT} available installation platforms
```

Comment:
For minimalism and to avoid annoying me, can you remove "available" from this heading? :)

Reply:
Since you asked nicely....